### ISSUE TITLE
make fails if no NOSYSHOSTKEYLOAD=1 is given

### ISSUE THREAD

**Issue by lez** [pts/pts-dropbear#3]
`make WRITEOPENSSHKEYS=1` results in the following error:
```
| ../dropbear-2017.75/svr-kex.c: In function 'svr_ensure_hostkey':
| 
| ../dropbear-2017.75/svr-kex.c:129:6: error: too few arguments to function 'signkey_generate'
|   if (signkey_generate(type, 0, fn, 1) == DROPBEAR_FAILURE) {
|       ^~~~~~~~~~~~~~~~
| 
| In file included from ../dropbear-2017.75/svr-kex.c:38:0:
| ../dropbear-2017.75/gensignkey.h:7:5: note: declared here
|  int signkey_generate(enum signkey_type type, int bits, const char* filename, int skip_exist, int format);
|      ^~~~~~~~~~~~~~~~
```
The build passes if NOSYSHOSTKEYLOAD=1 and WRITEOPENSSHKEYS=1 are both given to `make`.

**Comment by pts** [comment]
Thank you for reporting this! Fixed in 56632ca7b7a9869ba88afbfd40c68ec1e0562b41.

**Comment by lez** [comment]
Wow! Thanks for the instant fix! Really appreciate it!
